### DISCUSSION TITLE
nfcapd -x %u

### DISCUSSION THREAD

**Discussion by przemeksw** [phaag/nfdump#491] [General]
Can someone explain to me what the -x switch is really for and what command it triggers?
I can't understand it from the documentation :(
With nfcapd -x set to %u, I get the following entries in the logs:
Dec 16 12:15:00 nlogk nfcapd[29536]: Can't execvp: 1702725240: No such file or directory

Manual:
      -x command
              At the end of every -t interval and after the file rotate has completed, nfcapd runs the command command. The string for command may contain the following place
              holders, which are expanded before running:
              %f File name of new data file including any sub hierarchy.
              %d Top flowdir. The full path of the new file is: %d/%f
              %t Time slot string in ISO format e.g. 201107110845.
              %u Time slot string in UNIX time format.
              %i Identification string ident string supplied by -I

**Comment by phaag** [comment]
If you want to run a command at the end of each interval, this is the hook to use. 
If you want to feed the data to another database or if you want to move the file away in another location or whatever task you want to execute. Most of the time, users don't need this, some do.
Example: Move the new file to the advice directory:

```
nfcapd -S2 -t 60 -p 1234 -w /logs/vol0/ -D -j -B 128000 -x 'mv %d/%f /some/other/archive'
```

**Reply by przemeksw** [reply]
ok - I understand - thank you for the information
